client/ayon_core/hosts/substancepainter/plugins/load/load_mesh.py

```diff
@@ -24,49 +112,37 @@ class SubstanceLoadProjectMesh(load.LoaderPlugin):
     order = -10
     icon = "code-fork"
     color = "orange"
+    project_templates = []
 
-    options = [
-        qargparse.Boolean(
-            "preserve_strokes",
-            default=True,
-            help="Preserve strokes positions on mesh.\n"
-                 "(only relevant when loading into existing project)"
-        ),
-        qargparse.Boolean(
-            "import_cameras",
-            default=True,
-            help="Import cameras from the mesh file."
-        )
-    ]
-
-    def load(self, context, name, namespace, data):
+    def load(self, context, name, namespace, options=None):
 
         # Get user inputs
-        import_cameras = data.get("import_cameras", True)
-        preserve_strokes = data.get("preserve_strokes", True)
+        template_enum = [template["name"] for template
+                         in self.project_templates]
+        window = SubstanceProjectConfigurationWindow(template_enum)
+        window.exec_()
+        template_name = window.template_name
+        import_cameras = window.import_cameras
+        preserve_strokes = window.preserve_strokes
+        template = get_template_by_name(template_name, self.project_templates)
         sp_settings = substance_painter.project.Settings(
-            import_cameras=import_cameras
+            normal_map_format=_convert(template["normal_map_format"]),
+            project_workflow=_convert(template["project_workflow"]),
+            tangent_space_mode=_convert(template["tangent_space_mode"]),
+            default_texture_resolution=template["default_texture_resolution"]
         )
```

Comment:
Can we move the variables only relevant to the `if` statement into the `if` and those only used by the `else` into the `else`?